src/main/java/com/ericsson/ei/handlers/StatusHandler.java

```diff
@@ -0,0 +1,139 @@
+/*
+   Copyright 2019 Ericsson AB.
+   For a full list of individual contributors, please see the commit history.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package com.ericsson.ei.handlers;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import com.ericsson.ei.status.Status;
+import com.ericsson.ei.status.StatusData;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/**
+ * This class updates the statuses of EI, RabbitMQ and MongoDB on a given timer interval. When
+ * requested it returns status as a JsonNode.
+ *
+ */
+@Component
+public class StatusHandler {
+
```

Comment:
I'm not a fan of the word "handler". If possible, i think it could use another name representing what it does. In this case i think it could be resolved to "BackendStatus" class only. Then with the api it will give a great context. BackendStatus.getCurrentStatus() for example. The folder that it is in already states that it is an handler. :)

Reply:
Does Back-End know it is a Back-End? I do not like to use Back-End in the back-end.

Reply:
Fair enough, but i think you still can find a more suitable name. I dont know, SurroundingServicesStatus or something, if you find something better, please go a head and use that 😄